core/server/fluxruntime.go

```diff
@@ -164,18 +164,18 @@ func (cs *coreServer) GetReconciledObjects(ctx context.Context, msg *pb.GetRecon
 	var opts client.MatchingLabels
 
 	switch msg.AutomationKind {
-	case pb.FluxObjectKind_KindKustomization:
+	case "Kustomization":
```

Comment:
This is why it felt weird to me to delete `FluxObjectKind` all together, since if I do a typed thing here it has to match on the frontend request. If the type field on objects is `string` and not `Kind` in the frontend this will not work.......I think? At least that's what I thought was going on while I worked on this. I guess I can convert the string going in into this type and add a slot for "other" or something?